lib/src/cache/normalized_in_memory.dart

```diff
@@ -2,7 +2,7 @@ import 'package:meta/meta.dart';
 import 'package:graphql_flutter/src/utilities/traverse.dart';
 import 'package:graphql_flutter/src/cache/in_memory.dart';
 
-typedef String DataIdFromObject(Object node);
+typedef DataIdFromObject = String Function(Object node);
```

Comment:
I personally prefer the `typedef String DataIdFromObject(Object node);` style, but don't feel strongly. Everything else looks great IMO

Reply:
ah I see, [`- prefer_generic_function_type_aliases`](https://github.com/dart-lang/sdk/blob/master/docs/language/informal/generic-function-type-alias.md)